test.ice

```diff
@@ -1,6 +1,10 @@
 
 // This is a file ending in `.ice`.
 
+enum MyEnum {
+    A, B, C
+}
+
 module Hello::There {
```

Comment:
If you look at the file, it is correctly highlighted.
But, if you make a suggestion, the highlighting is completely broken:
```suggestion
module Hello::Theres {
```